starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -2639,7 +2638,7 @@ sub check_big_box_stars{
         }
     }
     if ($big_box_count < 3){
-        push @{$self->{warn}}, "$alarm Fewer than 3 ACQ stars with boxes >= 160 arcsec\n";
+        push @{$self->{yellow_warn}}, "$alarm Fewer than 3 ACQ stars with boxes >= 160 arcsec\n";
```

Comment:
Should I just chuck this warning altogether for starcheck 13.0?  I thought we might want training wheels for one release.

Reply:
Chuck it.